### PR TITLE
basic wrath support

### PR DESCRIPTION
**Pull request by jelmerj**
Just helping out with making a proper pull request. This is not my code, it's Makena's

### CODE REVIEW

File: ItemRack/ItemRack.lua [hunk 1]
```diff
@@ -382,11 +386,7 @@ end
 function ItemRack.UpdateClassSpecificStuff()
 	local _,class = UnitClass("player")
 
-	if class=="WARRIOR" or class=="ROGUE" or class=="HUNTER" or class=="MAGE" or class=="WARLOCK" then
-		ItemRack.CanWearOneHandOffHand = 1
-	end
-
-	if class=="SHAMAN" then
+	if class=="WARRIOR" or class=="ROGUE" or class=="HUNTER" or class=="MAGE" or class=="WARLOCK" or class=="SHAMAN" or class=="DEATHKNIGH" then
```
Comment (Rottenbeer) [inline ItemRack/ItemRack.lua:389]: Sure it's not DEATHKNIGHT?

**Review by Rottenbeer — CHANGES_REQUESTED** [review]

File: ItemRack/ItemRack.lua (hunk 1 above)
Comment (jelmerj) [reply ItemRack/ItemRack.lua]: doh, ofcourse it is

**Comment by Rottenbeer** [comment]
Thanks!
Please merge 

[simplify code](https://github.com/Rottenbeer/ItemRack/pull/203/commits/380a598a7d81d108bc50031cac467b5b30eef0d7)
[fix lua mistake](https://github.com/Rottenbeer/ItemRack/pull/203/commits/a7bd7db10719092f6d707fe49fd3dafb0f56f9cc)

[fixed typo](https://github.com/Rottenbeer/ItemRack/pull/203/commits/ad52956f0edc104e4d31f086ee09dd3574704ac5)

into a single commit and if you want to be extra nice, create a single commit for the two event lines in this commit:
https://github.com/Rottenbeer/ItemRack/pull/203/commits/d971b8c851c9f87f66383a083efb818766dd9f8f#diff-7b6cfe7e5c28cdcb03d17549b5dac9e194b6d987978a281e4755417afee86868R488

If you don't want to fiddle with git, I can do it also on my own ;)

Afterwards I'd merge it

**Comment by jelmerj** [comment]
I'm definitely not a Git Ninja but I can follow a guide :)
I'm not sure this is the outcome you were expecting though, I would have expected the old commits to disappear from my commit history.

**Comment by Rottenbeer** [comment]
haha :D that's not how to do it.
Just do a git rebase -i HEAD~8 and you should see your editor and a list of commits like that:

```
pick 860c383 Release 3.66
pick 0319937 Add last bank slot for TBC
pick 8985dbb Release 3.67
pick 11c4e4f Move Bindings to PLAYER_ENTERING_WORLD
pick e810bdb Add Tree of Life event
pick 4be2174 Release 3.68
pick c726986 Fix lua erros with wow classic 1.14
pick d9f85da Release-3.69
```

the latest commit is at the bottom, the oldest one at the top.
You need to read this list from the bottom up.
You want to squash the commits I meantioned now into a single commit, so change the "pick" in front of those, to a "s" and exit your editor.
Let's say I want to squash the last three commits in to "Add Tree of Life event", then I have to do it like that:
```
pick 860c383 Release 3.66
pick 0319937 Add last bank slot for TBC
pick 8985dbb Release 3.67
pick 11c4e4f Move Bindings to PLAYER_ENTERING_WORLD
pick e810bdb Add Tree of Life event
s 4be2174 Release 3.68
s c726986 Fix lua erros with wow classic 1.14
s d9f85da Release-3.69
```

In your case, you can also add a "d" in front of the merge commit, because we don't need this anymore.

**Comment by Rottenbeer** [comment]
> 

Do you have a warrior or a wow subscription and can confirm that the change works?

**Comment by jelmerj** [comment]
I do have an active subscription, and yes, the changes work on WotLK. I do not have a warrior but the titansgrip changes or in the other pull request so that should be out of scope for this PR.
I have not tested this on regular classic or tbc classic, as I dont have any chars on those servers.

**Review by Rottenbeer — APPROVED** [review]

**Pull request merged**